features/validate_image.feature

```diff
@@ -81,10 +81,8 @@ Feature: evaluate enterprise contract
           "name": "Unnamed",
           "containerImage": "localhost:(\\d+)/acceptance/invalid-image-signature",
           "violations": [
-            {"msg": "no matching signatures:\ninvalid signature when validating ASN.1 encoded signature"},
-            {"msg": "no matching attestations:\nAccepted signatures do not match threshold, Found: 0, Expected 1"},
-            {"msg": "EV001: No attestation data, at github.com/hacbs-contract/ec-cli/internal/evaluation_target/application_snapshot_image/application_snapshot_image.go:47"},
-            {"msg": "no attestations available"}
+            {"msg": "No images signatures found matching the given public key. Verify the correct public key was provided, and a signature was created."},
```

Comment:
Is it supposed to be `image signatures` or `images' signatures` maybe, with a possessive apostrophe?

Reply:
I think image signatures is the terminology I've seen used elsewhere.